app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = Foods::IndexFacade.search_by_food(params[:id])
+  end
+end
```

Comment:
where is this `:id` parameter coming from? The form that you inherited should have the search parameter coming in with the key of `:q`.